tensorflow/core/kernels/mkl_conv_grad_input_ops.cc

```diff
@@ -523,7 +524,7 @@ class MklConvCustomBackpropInputOp
     // Conv[2D|3D]BackpropInputV2 supports both DT_INT32 and DT_INT64
     // output_shape tensor::MakeShape is able to handle both DT_INT32 and
     // DT_INT64 for input_tensor.
-    CHECK_EQ(this->MakeShape(input_tensor, &input_tf_shape).ok(), true);
+    CHECK_EQ(tensor::MakeShape(input_tensor, &input_tf_shape).ok(), true);
```

Comment:
We got a `no new check in code` error. Could you please change this to `DCHECK_EQ` instead? Thank you!
```suggestion
    DCHECK_EQ(tensor::MakeShape(input_tensor, &input_tf_shape).ok(), true);
```

Reply:
Done.